python/kfserving/server.py

```diff
@@ -53,7 +53,7 @@ def start(self, models=[]):
             (r"/models/([a-zA-Z0-9_-]+):predict",
              ModelPredictHandler, dict(models=self.registered_models)),
         ]))
-        
```

Comment:
Which linter are you using? Somehow my pylint didnt catch this.

Reply:
I'm using builtin with pyCharm